js/build.js

```diff
@@ -30,6 +30,12 @@ Fliplet.Widget.instance('list-thumb-l', function(data) {
     }
   });
 
+  $container.on('click', '.filter-' + data.id, function(event) {
+    var dataset = event.target.dataset;
+
+    $container.toggleClass('saved-list', dataset.filter && dataset.filter === '.saved');
```

Comment:
Can you verify under what scenario is the first `dataset.filter` needed?

```suggestion
    $container.toggleClass('saved-list', dataset.filter === '.saved');
```

Reply:
I added it because thought that we need to ensure that we have `dataset.filter` but all `fliter-` elements has it. Fixed.